mlflow/store/artifact/azure_data_lake_artifact_repo.py

```diff
@@ -208,15 +208,14 @@ def _multipart_upload(self, credentials, src_file_path, artifact_file_path):
                 headers=headers,
             )
             # next try to append the file
-            futures = {}
             file_size = os.path.getsize(src_file_path)
-            num_chunks = _compute_num_chunks(
-                src_file_path, MLFLOW_MULTIPART_UPLOAD_CHUNK_SIZE.get()
-            )
+            chunk_size = MLFLOW_MULTIPART_UPLOAD_CHUNK_SIZE.get()
+            num_chunks = _compute_num_chunks(src_file_path, chunk_size)
             use_single_part_upload = num_chunks == 1
-            for index in range(num_chunks):
-                start_byte = index * MLFLOW_MULTIPART_UPLOAD_CHUNK_SIZE.get()
-                future = self.chunk_thread_pool.submit(
+
+            def upload_fn(index):
+                start_byte = index * chunk_size.get()
```

Comment:
Do we need the 2nd `.get()` on this scalar?

Reply:
No, thanks for the catch!